src/web/client/common/portalSchemaReader.ts

```diff
@@ -0,0 +1,32 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { portal_schema_data } from "./portalSchema";
+
+export function readSchema() {
+    return getOrganizationMap()
+}
+
+export function getSchemaMapForEntities() {
+    const organizationMap = new Map<string, string>();
+    for (const pair in portal_schema_data.entities.entity.entries()) {
+        console.log(pair[0], "key, value", pair[1])
+        organizationMap.set(pair[0], pair[1])
+    }
+    return organizationMap
+}
+
+export function getOrganizationMap() {
```

Comment:
Depending on the name of the Organizations object in the Schema, please rename these.